op-batcher/batcher/channel_manager.go

```diff
@@ -29,7 +28,6 @@ type ChannelOutFactory func(cfg ChannelConfig, rollupCfg *rollup.Config) (derive
 // channel.
 // Public functions on channelManager are safe for concurrent access.
 type channelManager struct {
-	mu          sync.Mutex
```

Comment:
This mutex has been removed in favor of one in the parent struct (the driver), so that it can have greater control over which operations to drive atomically.